packages/pharos/src/components/alert/pharos-alert.ts

```diff
@@ -52,9 +47,10 @@ export class PharosAlert extends FocusMixin(LitElement) {
   /**
    * The status to reflect to the user
    * @attr status
+   * @type {AlertStatus | undefined}
    */
   @property({ type: String, reflect: true })
-  public status: AlertStatus = '';
+  public status?: AlertStatus;
```

Comment:
We have `strict` on for TypeScript which includes [Strict Class Initialization](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#strict-class-initialization). We could use the definite assignment assertion modifier but it should only be used for things we know exist. Without explicitly saying it can be undefined some other function/s could throw an error (such as `_getIcon`) and make it harder for folks to debug the underlying issue.

Reply:
Right—I guess my question was basically whether we would need to worry about this if we only served TypeScript projects, since we could force a compilation error if the `status` wasn't supplied.

Reply:
Perhaps we should use the definite assignment assertion modifier to avoid confusion for TypeScript users.

Reply:
As long as we have tests for the expected error to be thrown we should be covered from any risks.